src/useEditor.ts

```diff
@@ -105,7 +105,7 @@ const useEditor = (initialState: EditorState, createView: CreateView) => {
         if (node.attrs.id === id) {
           currentView.focus()
 
-          const selection = node.isAtom
+          const selection = !node.isAtom
```

Comment:
I'm not fully convinced by this change. Could you please provide more details on how this could resolve the problem without affecting the existing functionality?

Reply:
@IslamJMomani I will use just NodeSelection, unless there is a reason we use TextSelection!